oblib/parser.py

```diff
@@ -14,10 +14,8 @@
 
 """ Parses JSON/XML input and output data. """
 
-import data_model
-import taxonomy
-import util
-
+from .data_model import OBInstance, Context
```

Comment:
does `from oblib import data_model` work?  That would preserve the module name in the namespace, i.e., `data_model.OBInstance`